### DISCUSSION TITLE
Hi perhaps a silly question but.....

### DISCUSSION THREAD

**Discussion by Ramz1337** [Digitalroot-Valheim/Digitalroot.Valheim.MaxDungeonRooms#5] [Q&A]
Fixed minor error

**Comment by Digitalroot** [comment]
>on a Windows Dedicated Valhiem Server Installed your outstanding mod to increase the Dungeon sizes however within BepInex when you press F1 you cannot adjust the slider or adjust the max / min for the dungeon sizes.
>
>I could however from the client end you can in which I tried and when in the Server the slider bar still shows the default values.
>
>hat am I doing wrong as in how do you set the dungeon sizes if not through the admin control panel on BepInex as that does not register anything from the client?
>
>Sorry if stupid question :)

I've never tried to config the DS w/ F1. Is that a thing?

**Reply by Ramz1337** [reply]
Hi when on the Server and logged in as admin who can press F1 and it brings up a console where you can change your Mod Settings.

I managed top sort the slider bar while on the Server logged in from a client. Is this where you expect to set the sizes or is there a file we edit?

Please see screenshot
![v1](https://user-images.githubusercontent.com/98008544/150338333-28311adf-b52d-4393-8cad-19b5ec5b9c66.JPG)

I have now gone into several dungeons including new discovery in areas not explored and the dungeon size still looks standard max 40. As you can see in the min rooms is 93 and I am certainly not seeing that.

Sorry if its a dumb question we normally play on a default server but installed this mod as it will bring a refresh approach and game play for me and my friends 

Thanks in advance.

**Reply by Digitalroot** [reply]
I think all that does is changed the values for your local client. I do not think they get written to the server. In `Valheim\BepInEx\config` is file named `digitalroot.mods.maxdungeonrooms.cfg`. This config file is on both the clients and the server. When the client starts up, it uses the values from the local config. Once connected to the server, the server pushes it's values to the client. At that point the client uses the values from the sever. It does this w/o overwriting the local config file. 

See https://github.com/Digitalroot-Valheim/Digitalroot.Valheim.MaxDungeonRooms#the-algorithm for what the Min/Max values actually do. They do not work as you think they do. I also miss-understood them. 

I suggest you set both values to 1000 and see if that helps you get larger room counts. @ 1000 the most I got was like 99 rooms and the least I got was 6.

**Reply by Ramz1337** [reply]
Thanks I have done as you have suggested and Ill feed back the success etc. Two last question you mention Dungeons only get created when you get near ( what is the definition of near?)  and the last I have installed as you can see Dungeon Reset in conjunction with your Mod would you know if reset they would then use your Mod / control recreate a new Dungeon or just reset with mobs?

Thanks again for a Brill Mod much appreciated

**Reply by Digitalroot** [reply]
@Ramz1337 Sorry I missed your last question in there. It depends on how the reset works. If it is just setting everything as not dead/opened, then my mod will not help. 

If the reset is causing the dung to be regened, then my mod will help. You can test this by resetting a dung and check if the layout has changed. If it is the same layout, then it is not being regened.
